pkg/store/container/container_test.go

```diff
@@ -99,12 +110,22 @@ func TestContainerStore(t *testing.T) {
 			Message:    "TestMessage-3",
 			Removing:   true,
 		},
+		"4abcd": {
+			Pid:        4,
+			CreatedAt:  time.Now().UnixNano(),
+			StartedAt:  time.Now().UnixNano(),
+			FinishedAt: time.Now().UnixNano(),
+			ExitCode:   4,
+			Reason:     "TestReason-4abcd",
+			Message:    "TestMessage-4abcd",
+			Removing:   true,
+		},
 	}
 	assert := assertlib.New(t)
 	containers := map[string]Container{}
-	for _, id := range ids {
+	for id, v := range metadatas {
 		container, err := NewContainer(
-			metadatas[id],
+			v,
```

Comment:
nit: Can we still use `metadatas[id]`?

So that in the future, if this becomes a pointer, this still won't have problem

Reply:
done